python/cac_tripplanner/destinations/views.py

```diff
@@ -38,11 +40,12 @@ def home(request):
     # Load one random article
     article = Article.objects.random()
     # Show all destinations
-    destinations = Destination.objects.published().all()
+    destinations = list(Destination.objects.published().all().order_by('priority'))
+    events = list(Event.objects.current().all().order_by('priority', 'start_date'))[:2]
```

Comment:
Super minor, but the `.all()` calls don't do anything here, which makes them slightly de-clarifying.